### PR TITLE
updated Footer.tsx to always display the correct year

### PR DESCRIPTION
**Pull request by greencarlos**
Update the /components/Footer.tsx file to always have the current year
Rather than hard code the correct year January 1st can't we just use `String(date).split(' ')[3]`

### CODE REVIEW

**Comment by greencarlos** [comment]
Footer tsx

**Comment by SlyBouhafs** [comment]
Looks like the preview deploy is failing, your branch is outdated, can you get you update your branch with the latest updates from upstream?

**Comment by greencarlos** [comment]
> @greencarlos Looks like the preview deploy is failing, your branch is outdated, can you update your branch with the latest changes from upstream?

The branch should be more updated at this time
Let me know if anything else is missing

File: components/Footer.tsx [hunk 1]
```diff
@@ -9,7 +9,9 @@ const Footer: React.FC<Props> = ({ footerType }) => {
     ? `fw-light text-center ${footerType}`
     : `mt-5 fw-light text-center pb-5 text-muted`
   return (
-    <footer className={footerClass}>&copy; Copyright 2022 GarageScript</footer>
+    <footer className={footerClass}>
+      &copy; Copyright {String(new Date()).split(' ')[3]} GarageScript
```
Comment (bryanjenningz) [inline components/Footer.tsx:13]: I think `new Date().getFullYear()` would be more clear.

Comment (greencarlos) [reply components/Footer.tsx:13]: Making the change now

File: README.md [hunk 2]
```diff
@@ -72,6 +74,9 @@ Thanks goes to these wonderful people
     <td align="center"><a href="https://github.com/flacial"><img src="https://avatars.githubusercontent.com/u/35906419?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Flacial</b></sub></a><br /><a href="https://github.com/garageScript/c0d3-app/commits?author=flacial" title="Code">💻</a></td>
     <td align="center"><a href="https://github.com/HS-90"><img src="https://avatars.githubusercontent.com/u/77421872?v=4?s=100" width="100px;" alt=""/><br /><sub><b>HS-90</b></sub></a><br /><a href="https://github.com/garageScript/c0d3-app/commits?author=HS-90" title="Code">💻</a></td>
   </tr>
+    <td align="center"><a href="https://github.com/greencarlos"><img src="https://avatars.githubusercontent.com/u/30880055?v=4" width="100px;" alt=""/><br /><sub><b>Carlos Green</b></sub></a><br /><a href="https://github.com/garageScript/c0d3-app/commits?author=greencarlos" title="Code">💻</a></td>
```
Comment (SlyBouhafs) [inline README.md:77]: Can you please omit these changes? The contributors are actually added via a bot and not manually.

**Comment by SlyBouhafs** [comment]
> > @greencarlos Looks like the preview deploy is failing, your branch is outdated, can you update your branch with the latest changes from upstream?
> 
> The branch should be more updated at this time Let me know if anything else is missing

Actually, your branch is way behind the master upstream branch, what you should do is switch to the `master` branch on your fork, merge changes from upstream `master` into your fork, then merge those changes into your `moneymakingmitch` branch.

File: README.md (hunk 2 above)
Comment (greencarlos) [reply README.md]: Done deal

**Review by bryanjenningz — APPROVED** [review]

**Pull request merged**